tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/SupplierSectionSteps.cs

```diff
@@ -160,6 +162,20 @@ public async Task ThenTheSupplierForOrderIsUpdated(string orderId, Table table)
             actual.Should().BeEquivalentTo(supplier);
         }
 
+        [Given(@"the user wants to update the supplier address section")]
+        public void WhenTheUserWantsToUpdateTheSupplierAddressSection(Table table)
+        {
+            var address = table.CreateSet<SupplierAddressTable>().FirstOrDefault();

```

Comment:
This could use `table.CreateInstance<SupplierAddressTable>()` rather than `CreateSet`. This would mean we don't need the `FirstOrDefault` call.

Reply:
👍